data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Stormhold Artifacts [1] Coded Message"
+	name "Go to <planet>"
+	description `Head to <planet> to deal with a "security issue."`
+	landing
+	to offer
+		has "event: stormhold korath issue"
+		not "Hai Reveal [B12] Devil-Hide Battle: offered"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai" "Hai (Unfettered)" "Pirate"
+		not planet Stormhold
+	destination Stormhold
+	on offer
+		conversation
+			`As soon as you land, your communications console beeps urgently. It's a coded message from <planet> that simply reads, "Security issue. Return to <planet> as soon as possible."`
+			`	The digital signature claims to be of diplomatic origin, and from the burning acorn insignia, it is most likely from the Unfettered.`
+				accept
+
+mission "Korath Stormhold Artifacts [2] Drones"
+	landing
+	name "Destroy Korath drones at <waypoints>."
+	description `Destroy Korath stealth drones at <waypoints>. This is an optional mission; an elite Navy group will do it if you don't.`
```

Comment:
```suggestion
	description `Destroy Korath stealth drones at <waypoints>. This is an optional mission; an elite Navy group will do it if you choose not to.`
```

Reply:
The navy will also finish the mission if you die. Your choice is less important to them than your success.